concourse/scripts/smoke_tests_email.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+set -eu
+
+DEPLOY_ENV=$1
+SYSTEM_DNS_ZONE_NAME=$2
+ALERT_EMAIL_ADDRESS=$3
+
+TO="${ALERT_EMAIL_ADDRESS}"
+FROM="${ALERT_EMAIL_ADDRESS}"
```

Comment:
> Or do we want this to be a real address that we can reply to?

It should be a real address. Not necessarily so that we can reply to it, but to make it easier to understand where the emails have come from. `@digital.cabinet-office.gov.uk` is used by a lot of people and projects.